test/lib/y2erb_test.rb

```diff
@@ -22,11 +22,179 @@
+        "bus_hwcfg"         => "pci",
+        "bus_id"            => 2,
+        "class_id"          => 2,
+        "dev_name"          => "enp2s0",
+        "dev_names"         => ["enp2s0"],
+        "device"            => "RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller",
+        "device_id"         => 98664,
+        "driver"            => "r8169",
+        "driver_module"     => "r8169",
+        "drivers"           => [{ "active" => true, "modprobe" => true,
+        "modules" => [["r8169", ""]] }],
+        "modalias"          => "pci:v000010ECd00008168sv00001043sd0000208Fbc02sc00i00",
+        "model"             =>
+                               "Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller",
+        "old_unique_key"    => "bbxe.g1yJ3amPfwB",
+        "parent_unique_key" => "e6j0.CB0G3oMQsS4",
+        "resource"          => {
+          "hwaddr"  => [{ "addr"=>"a8:5e:45:c1:2f:eb" }],
+          "io"      => [{ "active" => true, "length" => 256,
+                                                            "mode" => "rw", "start" => 57344 }],
+          "irq"     => [{ "count" => 0, "enabled" => true,
+                                                            "irq" => 71 }],
+          "link"    => [{ "state"=>false }],
+          "mem"     =>
+                       [{ "active" => true, "length" => 4096,
+                                                                         "start" => 4152377344 },
+                        { "active" => true, "length" => 16384,
+                        "start" => 4152360960 }],
+          "phwaddr" => [{ "addr"=>"a8:5e:45:c1:2f:eb" }]
+        },
+        "rev"               => "21",
+        "sub_class_id"      => 0,
+        "sub_device_id"     => 73871,
+        "sub_vendor"        => "ASUSTeK Computer Inc.",
+        "sub_vendor_id"     => 69699,
+        "sysfs_bus_id"      => "0000:02:00.0",
+        "sysfs_id"          => "/devices/pci0000:00/0000:00:01.2/0000:02:00.0",
+        "unique_key"        => "c3qJ.g1yJ3amPfwB",
+        "vendor"            => "Realtek Semiconductor Co., Ltd.",
+        "vendor_id"         => 69868
+      },
+      {
+        "bus"               => "PCI",
+        "bus_hwcfg"         => "pci",
+        "bus_id"            => 4,
+        "class_id"          => 2,
+        "device"            => "RTL8821CE 802.11ac PCIe Wireless Network Adapter",
+        "device_id"         => 116769,
+        "modalias"          => "pci:v000010ECd0000C821sv00001A3Bsd00003041bc02sc80i00",
+        "model"             => "Realtek RTL8821CE 802.11ac PCIe Wireless Network Adapter",
+        "old_unique_key"    => "aher.1HJgn3FRa50",
+        "parent_unique_key" => "yk9C.CB0G3oMQsS4",
+        "resource"          =>
+                               { "io"  =>
+                                          [{ "active" => false, "length" => 256, "mode" => "rw",
+                                                             "start" => 53248 }],
```

Comment:
The indentation looks wrong (same below).

Reply:
well, I use just pp in irb and then let rubocop reformat it. Data is so huge, that it was not so easy to even read it.